### PR TITLE
Move export events to separate folder

### PR DESCRIPTION
**Pull request by nikitavemuri**
## Why are these changes needed?
- Move export events from `session_latest/logs/events` to `session_latest/logs/export_events`
- Keeping both event types in the same folder doesn't cause any issue for Ray -- export event files are already filtered out for `/events` API in  https://github.com/ray-project/ray/blob/1e48a033af1cbc7297cc9b129a3f0eb247239c5a/python/ray/dashboard/modules/event/event_utils.py#L22
- However moving these to a separate folder would be better for existing downstream consumers to avoid handling export events in the `events` folder if they turn the flag on

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: python/ray/tests/test_runtime_env_2.py [hunk 1]
```diff
@@ -240,7 +240,9 @@ def job_failed(job_id):
         with pytest.raises(AssertionError):
             assert_no_user_info_in_logs(USER_SECRET)
 
-        assert_no_user_info_in_logs(USER_SECRET, file_whitelist=["runtime_env*.log"])
+        assert_no_user_info_in_logs(
+            USER_SECRET, file_whitelist=["runtime_env*.log", "event_EXPORT*.log"]
```
Comment (alanwguo) [inline python/ray/tests/test_runtime_env_2.py:244]: why is this necessary? There shouldn't be "user info" in the export events is there?

Comment (nikitavemuri) [reply python/ray/tests/test_runtime_env_2.py:244]: Various export events contain runtime env info (which is considered user info) because the runtime env is returned in some dashboard APIs

**Review by alanwguo — APPROVED** [review]

**Review by rkooo567 — APPROVED** [review]
